### PR TITLE
Travis and auth cfg cleanup

### PR DESCRIPTION
**Pull request by gyorb**
* remove dummy authentication credentials in it enough it they are in the documentation
* cleanup configuration files in each travis run

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,5 +57,6 @@ addons:
     postgresql: "9.3"
 
 script:
+    - make clean_travis
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make travis; fi
     - make test
```
Comment (Xazax-hun) [inline .travis.yml]: I think we could also add a make clean at the end just to make sure that none of the scripts/command exit unexpectedly.

File: Makefile [hunk 2]
```diff
@@ -41,6 +41,10 @@ travis:
 	# travis specific changes
 	scripts/change_clang_version.py $(CURRENT_DIR)
 
+clean_travis:
+	# cleanup codechecker config files
```
Comment (Xazax-hun) [inline Makefile]: We should have full sentence comments in the makefile too.

File: .travis.yml (hunk 1 above)
Comment (gyorb) [inline .travis.yml]: I'm not sure what you mean here by the scripts/commands exit unexpectedly, but we can make some cleanup after the tests too.

File: Makefile (hunk 2 above)
Comment (gyorb) [inline Makefile]: Done.

**Pull request merged**